tests/math/line2d_tests.cc

```diff
@@ -92,6 +92,9 @@ TEST(Line2D, ClosestApproach) {
 
   EXPECT_FLOAT_EQ(Line2f(1, 1, 10, 1).ClosestApproach(
       Vector2f(-100, 0), Vector2f(-1, -1)), sqrt(8.0f));
+
```

Comment:
Unit tests that check for calculations must check for equality with hand-computed values, not inequalities.